dashboard/src/modules/components/AuthComponent/common-components.jsx

```diff
@@ -85,6 +85,9 @@ export const AuthForm = () => {
   const navigatePage = (toPage) => {
     dispatch(movePage(toPage, navigate));
   };
+  useEffect(() => {
+    dispatch(loadTokens());
+  }, );
```

Comment:
Are you supposed to be specifying a second argument to `useEffect()` (i.e., something after that comma), like a capture, perhaps of the value of `dispatch`?

Otherwise, would this work?
```
  useEffect(() => dispatch(loadTokens()));
```

Reply:
Removed the `loadTokens`